src/app/shared/services/iam.service.ts

```diff
@@ -167,4 +172,12 @@ export class IamService {
 
     return retVal;
   }
+
+  private observableWrapper<T>(source: Promise<T> | Observable<T>,
+                               loaderConfig?: { message?: string | string[]; cancelable?: boolean }) {
+    this.loadingService.show(loaderConfig.message, !!loaderConfig.cancelable);
```

Comment:
If asserting with `!!` that `cancelable` is defined, wouldn't it better to require that in the param?
```suggestion
                               loaderConfig?: { message?: string | string[]; cancelable: boolean }) {
    this.loadingService.show(loaderConfig.message, loaderConfig.cancelable);
```

Reply:
No, because not always it is required and not always defined.